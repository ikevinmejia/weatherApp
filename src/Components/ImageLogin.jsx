import React from 'react'
import imgLogin from '../assets/img/login.svg'

const ImageLogin = () => {
  return (
    <figure className='w-full h-1/3 lg:w-1/2 lg:h-2/5'>
        <img src={imgLogin} alt="" className='w-full h-full object-contain' />
    </figure>
  )
}

export default ImageLogin