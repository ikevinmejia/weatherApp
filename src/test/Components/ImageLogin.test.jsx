import { render, screen } from "@testing-library/react"
import ImageLogin from "../../Components/ImageLogin"

describe('test in <ImageLogin/>', () => {

    
    test('math to snapshot', () => {
        const {container} = render(<ImageLogin/>)

        expect({container}).toMatchSnapshot()
    })

    test('Deben haber 3 tags', () => {
        render(<ImageLogin/>)

        expect(screen.getByTestId('test-figure')).toBeTruthy()
    })
})