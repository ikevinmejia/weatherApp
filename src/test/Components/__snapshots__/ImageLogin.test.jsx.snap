// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test in <ImageLogin/> math to snapshot 1`] = `
Object {
  "container": <div>
    <figure
      class="w-full h-1/3 lg:w-1/2 lg:h-2/5"
      data-testid="test-figure"
    >
      <img
        alt=""
        class="w-full h-full object-contain"
        src="function SvgLogin(props) {
  return React.createElement(
    'svg', 
    Object.assign({}, props, {'data-file-name': SvgLogin.name})
  );
}"
      />
    </figure>
  </div>,
}
`;
